main.cpp

```diff
@@ -0,0 +1,190 @@
+#include <iostream>
+#include <fstream>
+
+// Location of config txt file
+const std::string DEFAULT_FILE_PATH = "/Users/naomibrown/assignments/Assignment2/MyConfigFile.txt";
+
+int main(int argc, char *argv[]) {
+    // Before doing any processing, check to see if there are sufficient arguments entered
+
+    if(argc<2){
+        std::cout << "You must enter at least one argument to run this program.\n";
+        return -1;
+    }
+
+    // Declare/initialize variables used in IF and ELSE IF statements
+    std::string firstName = "";
+    std::string lastName = "";
+    std::string emailAddress = "";
+    std::string password = "";
+    std::string timezoneOffset = "";
+    std::string knownrecipientsFilePath = "";
```

Comment:
Sorry - old habits die hard :)  I will need more practice for the concept of declaring variables immediately before they are used to become automatic...